src/learner.cc

```diff
@@ -300,6 +302,65 @@ class LearnerImpl : public Learner {
     this->configured_ = true;
   }
 
+  void Save(Json* p_out) const override {
+    // FIXME(trivialfis); num_boosting_round
```

Comment:
@hcho3 I forgot why we decided to save `num_boosted_rounds` (number of training iterations done so far) before.  Could you remind me a little?

Reply:
@trivialfis See https://xgboost-json-schema.readthedocs.io/en/latest/#gbtree

Reply:
@hcho3 Yes I see that it's there, but I don't remember why it's there.  Currently it's not save and XGBoost doesn't seem to have any problem from recovering.

Reply:
@trivialfis I think it's because we store the array of tree objects, and the DMLC serializer for arrays stores the length of the array.

Reply:
@hcho3 I will remove it in the schema then.